enterprise/frontend/src/metabase-enterprise/whitelabel/components/HelpLinkSettings/HelpLinkRadio.tsx

```diff
@@ -0,0 +1,79 @@
+import { t } from "ttag";
+import { useState } from "react";
+import { Radio, Stack } from "metabase/ui";
+import type { HelpLinkSetting, SettingKey, Settings } from "metabase-types/api";
+import InputBlurChange from "metabase/components/InputBlurChange";
+
+interface Props {
+  setting: {
```

Comment:
TODO: the settings here are manually typed, I'll draft a proper typing later and ask for review/help to the web app team later